vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/io/parse/common/RuntimeResourceParseException.java

```diff
@@ -0,0 +1,16 @@
+package ca.bc.gov.nrs.vdyp.io.parse.common;
+
+public class RuntimeResourceParseException extends RuntimeException {
```

Comment:
This class appears to be unused. Which is a good thing, IMO, since I hate application level RuntimeExceptions.